doc/PIDHandler.md

```diff
@@ -0,0 +1,229 @@
+# PIDHandler introduction and usage
```

Comment:
Maybe somewhere it should say that it's not mandatory to use the PIDHandler and one can iterate manually? For example if each reco particle needs to be accessed once for a given PID algorithm, then it's not necessary to build a map.

Reply:
Good point. Done.